swift/Sources/TronAddress.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+//
+// This is a GENERATED FILE, changes made here WILL BE LOST.
+//
+
+import Foundation
+
+    public final class TronAddress {
+
+  public
```

Comment:
The formatting in this file sucks

Reply:
That was the clang-format commit hook 🤦‍♂️

Reply:
lol